src/explore-education-statistics-frontend/src/modules/find-statistics/components/Filters.tsx

```diff
@@ -17,45 +16,25 @@ export type FilterChangeHandler = ({
 
 interface Props {
   releaseType?: ReleaseType;
-  showMobileFilters: boolean;
   themeId?: string;
   themes: ThemeSummary[];
-  totalResults?: number;
   onChange: FilterChangeHandler;
-  onCloseMobileFilters: () => void;
 }
 
-const Filters = ({
-  releaseType,
-  showMobileFilters = false,
-  themeId,
-  themes,
-  totalResults,
-  onChange,
-  onCloseMobileFilters,
-}: Props) => {
+const Filters = ({ releaseType, themeId, themes, onChange }: Props) => {
```

Comment:
Perhaps it'd be good to be more specific and call these `FiltersDesktop`?

Would suggest also renaming `MobilesFilters` to `FiltersMobile` to match this so they end up next to each other in the project files.